pylint/config/find_default_config_files.py

```diff
@@ -24,40 +26,45 @@ def _cfg_has_config(path):
     return any(section.startswith("pylint.") for section in parser.sections())
 
 
-def find_default_config_files():
-    """Find all possible config files."""
-    rc_names = ("pylintrc", ".pylintrc")
-    config_names = rc_names + ("pyproject.toml", "setup.cfg")
+def _get_config_paths(curdir: Union[Path, str]) -> List[str]:
+    paths = []
+    config_names = ("pylintrc", ".pylintrc", "pyproject.toml", "setup.cfg")
     for config_name in config_names:
-        if os.path.isfile(config_name):
-            if config_name.endswith(".toml") and not _toml_has_config(config_name):
+        config_path = os.path.join(curdir, config_name)
+        if os.path.isfile(config_path):
+            if config_name.endswith(".toml") and not _toml_has_config(config_path):
                 continue
-            if config_name.endswith(".cfg") and not _cfg_has_config(config_name):
+            if config_name.endswith(".cfg") and not _cfg_has_config(config_path):
                 continue
 
-            yield os.path.abspath(config_name)
+            paths.append(config_path)
+
+    return paths
+
+
+def find_default_config_files():
```

Comment:
```suggestion
def find_default_config_files() -> str:
```

Reply:
I'm a newbie at type annotations, thanks for catch. :+1: